HHS/OPRE-OPS
# disable edit agreement UI based on status (for OPS-856)

## What changed

Disable agreement edit based on status

## Issue

#856 

## How to test

View the agreements list page and verify that the edit icon is not shown if the agreement is not DRAFT or UNDER_REVIEW.
Also if try to directly go to the edit page via URL like http://localhost:3000/agreements/edit/2, then editing will be disabled.
